lib/utils/whitespaceChecker.js

```diff
@@ -192,15 +195,12 @@ module.exports = function (targetWhitespace, expectation, messages) {
 	}
 
 	/**
-	 * @param {WhitespaceCheckerArgs} obj
+	 * @type {WhitespaceChecker}
 	 */
 	function beforeAllowingIndentation(obj) {
 		before({ ...obj, allowIndentation: true });
 	}
 
-	/**
-	 * @param {Function} [messageFunc]
-	 */
 	function expectBefore(messageFunc = messages.expectedBefore) {
```

Comment:
[note] `@param` is needless because `messages.expectedBefore` inferes a parameter type.